7forecast.tex

```diff
@@ -88,26 +87,26 @@
 
   \pagebreak
   % \hline
-  1 & Benchmark Relative F Basis: & \multirow{1}{1cm}[-0.2cm]{\parbox{12cm}{The specification does not affect year range for selectivity and biology.}} \Tstrut\\
+  1 & Benchmark Relative $F$ Basis: & \multirow{1}{1cm}[-0.2cm]{\parbox{12cm}{The specification does not affect year range for selectivity and biology.}} \Tstrut\\
     & 1 = use year range; and & \\
-    & 2 = set range for relF same as \hyperlink{Fcast}{Forecast}. & \Bstrut\\
+    & 2 = set range for $\text{rel}F$ same as \hyperlink{Fcast}{Forecast}. & \Bstrut\\
```

Comment:
Is "relF" used somewhere in the code base so you are tying to keep it consistent? I would just write out relative as a word.

Reply:
Yes it is used in 13output.tex